nixosModules/common.nix

```diff
@@ -35,58 +18,4 @@ in {
   nix.package = pkgs.nixUnstable;
   nix.requireSignedBinaryCaches = true;
   nix.settings.auto-optimise-store = true;
-  nix.settings.allowed-users = with config.users; [
```

Comment:
Please revert changes in this file, because it breaks dependencies https://github.com/profianinc/images/blob/a257ed235a5dd139026593c49767d9cfa18d571c/overlays/images.nix#L13